tests/foreman/ui/test_activationkey.py

```diff
@@ -1290,11 +1288,45 @@ def test_delete_manifest(self):
 
         @Assert: Deleting a manifest removes it from the Activation key
 
-        @Status: Manual
-
-        @BZ: 1078676
-
         """
+        # Upload manifest
+        manifest_path = manifests.clone()
+        entities.Organization(
+            id=self.org_id,
+        ).upload_manifest(path=manifest_path)
+        # Create activation key
+        activation_key = entities.ActivationKey(
+            organization=self.org_id,
+        ).create()
+        # Associate a manifest to the activation key
+        for subscription in \
+                entities.Organization(id=self.org_id).subscriptions():
```

Comment:
`entities.Organization(id=self.org_id)` is being used here and on the upload manifest. It is better to assign it to a variable.

If creating a new org:

```python
org = entities.Organization().create()
org.upoad_manifest(path=manifest_path)
# ...
for subscription in org.subscriptions():
# ...
```